src/core/ee/emotion.hpp

```diff
@@ -58,6 +61,8 @@ class EmotionEngine
         void print_state();
         void set_disassembly(bool dis);
 
+        Cop0* get_cop0();
+        Cop1* get_fpu();
```

Comment:
Recommend marking function as `const`.

```suggestion
        Cop1* get_fpu() const;
```